### PR TITLE
Feat/trending2

### PR DESCRIPTION
**Pull request by TonyGlimm**
Added trending API call, adjusted Movie class for it. changed naming so I can use same class for movies and tv-shows. wrote the test for the api call. put all of the features into these features to the frontend

### CODE REVIEW

File: backend/src/main/java/com/github/tonyglimm/media/Media.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+public record Media(
+        int id,
+        @JsonAlias("original_name") String original_title,
+        List<Integer> genre_ids,
+        String poster_path,
+        @JsonAlias("first_air_date") String release_date,
+        String media_type,
+        float vote_average,
+        int vote_count,
+        String overview) {
```
Comment (Kreyno93) [inline backend/src/main/java/com/github/tonyglimm/media/Media.java:15]: try to stick to the CamelCase convention

Comment (TonyGlimm) [reply backend/src/main/java/com/github/tonyglimm/media/Media.java:15]: Hmm. for the field names? Initially i named these like the fields in the JSON Object I recieve from the external API. Now that I know of @JSONAlias I could actually rename all of them to fields in CamelCase. Considering that do you still recommend renaming them and adding a @JsonAlias on each field?

Comment (Kreyno93) [reply backend/src/main/java/com/github/tonyglimm/media/Media.java:15]: use instead @JsonProperty

File: .github/workflows/deployToFly.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Build Frontend
         run: |
           cd frontend
-          npm install
+          npm install --force
```
[the remark this answers is not in the document]
Comment (Kreyno93) [reply .github/workflows/deployToFly.yml:19]: Steps:
1. delete MUI components in package json
2. delete react and react-dom
3. update react version // npm install react react-dom
4. install mui components with @latest annotation

**Review by Kreyno93 — APPROVED** [review]

**Pull request merged**